docs/install.md

```diff
@@ -1,5 +1,7 @@
 # Installation
 
+If PHP (>= 5.6) is not installed on your environment, install it now; PHP is a requirement for Dock CLI.
```

Comment:
Also php pecl SSH extension is needed, which isn't a standard extension people have installed